docusaurus/docs/learn/quickstarts/network/miniziti.bash

```diff
@@ -638,139 +744,143 @@ main(){
         | grep -q "public-routers"; then
         logDebug "creating service-edge-router-policy public-routers"
-        ziti edge create service-edge-router-policy "public-routers" \
+        ziti_wrapper edge create service-edge-router-policy "public-routers" \
             --edge-router-roles '#public-routers' --service-roles '#all' >&3
     else
         logDebug "ignoring service-edge-router-policy public-routers"
     fi
 
-    if [[ -s /tmp/httpbin-host.jwt ]]; then
-        logDebug "enrolling /tmp/httpbin-host.jwt"
+    if [[ -s "$HTTPBIN_OTT" ]]; then
+        logDebug "enrolling $HTTPBIN_OTT"
         # discard expected output that normally flows to stderr
         ENROLL_OUT="$(
-            ziti edge enroll /tmp/httpbin-host.jwt 2>&1 \
+            ziti edge enroll "$HTTPBIN_OTT" 2>&1 \
                 | grep -vE '(generating.*key|enrolled\s+successfully)' \
                 || true
         )"
         if [[ -z "${ENROLL_OUT}" ]]; then
-            rm -f /tmp/httpbin-host.jwt
-            logDebug "deleted /tmp/httpbin-host.jwt after enrolling successfully"
+            rm -f "$HTTPBIN_OTT"
+            logDebug "deleted $HTTPBIN_OTT after enrolling successfully"
         else
             echo -e "ERROR: unexpected result during OpenZiti Identity enrollment\n"\
                     "${ENROLL_OUT}"
             exit 1
         fi
     fi
 
-    if [[ -s /tmp/httpbin-host.json ]]; then
+    if [[ -s "$HTTPBIN_JSON" ]]; then
         logDebug "installing httpbin chart as 'miniziti-httpbin'"
         (( ZITI_CHARTS_ALT )) && {
             helm dependency build "${ZITI_CHARTS_REF}/httpbin" >&3
         }
         helm install "miniziti-httpbin" "${ZITI_CHARTS_REF}/httpbin" \
-            --set-file zitiIdentity=/tmp/httpbin-host.json \
+            --set-file zitiIdentity="$HTTPBIN_JSON" \
             --set zitiServiceName=httpbin-service >&3
-        rm -f /tmp/httpbin-host.json
-        logDebug "deleted /tmp/httpbin-host.json after installing successfully with miniziti-httpbin chart"
+        rm -f "$HTTPBIN_JSON"
+        logDebug "deleted $HTTPBIN_JSON after installing successfully with miniziti-httpbin chart"
     fi
 
-    kubectl get secrets "ziti-controller-admin-secret" \
-        --namespace "${ZITI_NAMESPACE}" \
-        --output go-template='{{"\n'\
-'INFO: Your OpenZiti Console is here:\thttps://miniziti-console.'"${MINIZITI_INGRESS_ZONE}"'\n'\
-'INFO: The password for \"admin\" is:\t"}}{{index .data "admin-password" | base64decode }}'\
-'{{"\n\n"}}'
+    echo -e "\n\n"
+    logInfo "Your OpenZiti Console is here: https://miniziti-console.${MINIZITI_INGRESS_ZONE}"
+    showAdminCreds
+    echo -e "\n\n"
 
-    logInfo "Success! Remember to add your edge client identity '$(getClientOttPath "${DETECTED_OS}")' in your tunneler, e.g. Ziti Desktop Edge."
+    logInfo "Success! Remember to add your edge client identity '$CLIENT_OTT' in your tunneler, e.g. Ziti Desktop Edge."
```

Comment:
This doesn't quite work the same in WSL because `getClientOttPath()` would print the resolved Windows UNC path by which the user could load the OTT in Ziti Desktop Edge for Windows. Now it prints the Linux path, which isn't useful for the Windows user that wants to connect to Ziti services provided by miniziti.

The vars built from the state dir are used at least two ways: one way provides paths for things like the `ziti edge create router` command writing the OTT file, and the other for Windows users to find the client OTT with a UNC path that was written to a POSIX path in WSL.

One way to handle this is to resurrect and modify `getClientOttPath()` and call it in this line. It could take the value of $CLIENT_OTT as input and return the same value if macOS or Linux and otherwise the resolved Windows UNC path.

Reply:
Good catch. I approached this in a slightly different, more generic way than your recommendation. Please re-test, and let me know if I got it right.

Reply:
Looks like it will work. I'll run in WSL this morning.

Reply:
This should be resolved.